src/App.js

```diff
@@ -0,0 +1,45 @@
+// import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Routes, Route } from 'react-router-dom';
+import Body from './components/Body';
+import Main from './components/Main';
+import SignIn from './components/SignIn';
+import SignUp from './components/SignUp';
+import AddDoctor from './components/AddDoctor';
+import Reservations from './pages/Reservations';
+import DoctorDetails from './pages/DoctorDetails';
+import DeleteDoctor from './components/DeleteDoctor';
+import AddReservation from './pages/AddReservation';
+import { fetchDoctors } from './redux/doctors';
+import { fetchReservations } from './redux/reservations';
+import Redirect from './pages/Redirect';
+
+function App() {
+  const dispatch = useDispatch();
+  // const doctors = useSelector((state) => state.doctors);
```

Comment:
- [ ] Kindly, remove this old commented code in order to improve the repo's cleanliness. 👍

Reply:
Thank you @Meltrust , we will remove it.